rocAL/include/api/rocal_api.h

```diff
@@ -50,9 +50,13 @@ THE SOFTWARE.
  * \param [in] cpu_thread_count number of cpu threads
  * \param [in] prefetch_queue_depth The depth of the prefetch queue.
  * \param [in] output_tensor_data_type RocalTensorOutputType: Defines whether the output of rocal tensor is FP32 or FP16.
+ * \param [in] last_batch_policy Describes the last batch policy behavior when there are not enough samples in the epoch to fill a whole batch.
+ * \param [in] last_batch_padded Whether the last batch provided by rocAL is padded with the last sample or it just wraps up.
+                                 In the conjunction with last_batch_policy it tells if the iterator returning last batch with 
+                                 data only partially filled with data from the current epoch is dropping padding samples or samples from the next epoch.
  * \return A \ref RocalContext - The context for the pipeline
  */
-extern "C" RocalContext ROCAL_API_CALL rocalCreate(size_t batch_size, RocalProcessMode affinity, int gpu_id = 0, size_t cpu_thread_count = 1, size_t prefetch_queue_depth = 3, RocalTensorOutputType output_tensor_data_type = RocalTensorOutputType::ROCAL_FP32);
+extern "C" RocalContext ROCAL_API_CALL rocalCreate(size_t batch_size, RocalProcessMode affinity, int gpu_id = 0, size_t cpu_thread_count = 1, size_t prefetch_queue_depth = 3, RocalTensorOutputType output_tensor_data_type = RocalTensorOutputType::ROCAL_FP32, std::pair<RocalLastBatchPolicy, bool> last_batch_info = {RocalLastBatchPolicy::ROCAL_LAST_BATCH_FILL, true});
```

Comment:
As I mentioned in the meeting, the last_batch stuff shouldn't be in context. It only affects the loading of data and hence it should be a parameter to appropriate reader/s

Reply:
Why is this still needed here?

Reply:
Removed the LastBatch parameters from rocalCreate and Pipeline creation